django/utils/http.py

```diff
@@ -33,6 +33,12 @@
 RFC3986_GENDELIMS = str(":/?#[]@")
 RFC3986_SUBDELIMS = str("!$&'()*+,;=")
 
+PROTOCOL_TO_PORT = {
+    'http': 80,
+    'https': 443,
+    'ftp': 21,
```

Comment:
But, yeah, I agree, I don't see why we would need anything other than http and https.